sdk/storage/azure-storage-files-datalake/inc/azure/storage/files/datalake/protocol/datalake_rest_client.hpp

```diff
@@ -2203,7 +2203,7 @@ namespace Azure { namespace Storage { namespace Files { namespace DataLake {
               ContentLength; // Required for "Append Data" and "Flush Data".  Must be 0 for "Flush
                              // Data".  Must be the length of the request content in bytes for
                              // "Append Data".
-          Azure::Core::Nullable<std::string> ContentMd5; // Specify the transactional md5 for the
+          Azure::Core::Nullable<ContentHash> ContentMd5; // Specify the transactional md5 for the
```

Comment:
Isn't this transactional MD5? So when do you use prefix `Transactional`, when don't?

Reply:
This is not, the comment is misleading due to a bug that I cannot easily resolve in my generator. However, this is not exposed to customers so my concern is minimum.

Reply:
No, it's not minimum. To me, the variable name is simply wrong.

Reply:
If we know it's a bug, we should fix it. This is the right thing we should do, isn't it?